3scale/porta
# Adds captcha to Dev portal Login

**What this PR does / why we need it**:

$Subject

**Which issue(s) this PR fixes** 

[THREESCALE-6645: Developer portal lack of brute force protection on login](https://issues.redhat.com/browse/THREESCALE-6645)

**Verification steps** 

1. Set spam protection 'Always' in Developer Portal > Spam protection
2. Go to the signup page in the dev portal. It should be required to pass the captcha
